manifests/i/iTopDataRecovery/iTopDataRecovery/3.0.0.177/iTopDataRecovery.iTopDataRecovery.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: iTopDataRecovery.iTopDataRecovery
+PackageVersion: 3.0.0.177
+PackageLocale: en-US
+Publisher: iTop
+PackageName: iTop Data Recovery
+License: © iTop Inc. All rights reserved.

```

A copyright isn't a license;
```suggestion
License: Proprietary
LicenseUrl: https://www.itopvpn.com/eula
Copyright: © iTop Inc. All rights reserved. 
```